tests/vars.sample.yaml

```diff
@@ -71,6 +71,9 @@ auth_url: http://keystone-public-openstack.apps-crc.testing
 # Set this to true if adopting the ironic services (ironic + ironic-inspector + nova w/compute-ironic)
 ironic_adoption: false
 
+# Name of network used in ironic spec networkAttachments and provisionNetwork
+ironic_network: baremetal
```

Comment:
In case we'd expect to often use the default and only customize the value in rare cases, it's ok to just add the variable into role's defaults and not add it into the vars.sample.yaml. I'm not sure if it's the case here and it's not a big issue, just mentioning it as a pattern that can be used in general.